openstack/cronus/templates/cronus/_config.yaml.tpl

```diff
@@ -108,26 +108,13 @@ cronus:
       errInterval: {{ .Values.cronus.tls.errInterval | default "60s" }}
 {{- end }}
   keystone:
-{{- if .Values.config.keystone }}
-{{- range $key, $value := .Values.config.keystone }}
```

Comment:
It looks like some of the values are config values. Would it make sense to list what it is? The loop was always a bit confusing to me, as it hides what the properties are.

For example
```
{{- if .Values.config.keystone }}
keystone:
    projectName: {{ .Values.config.projectName }}
    projectDomainName: {{ .Values.config.projectDomainName }}
    userDomainName:  {{ .Values.config.userDomainName }}
    username: {{ .Values.config.username }}
    ```